src/System.Runtime/tests/System/TypeTests.netstandard1.7.cs

```diff
@@ -0,0 +1,142 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using Xunit;
+
+namespace System.Tests
+{
+    public class TypeTestsExtended : RemoteExecutorTestBase
+    {
+        public class ContextBoundClass : ContextBoundObject
+        {
+            public string Value = "The Value property.";
+        }
+
+        static string sourceTestAssemblyPath = Path.Combine(Environment.CurrentDirectory, "TestLoadAssembly.dll");
+        static string destTestAssemblyPath = Path.Combine(Environment.CurrentDirectory, "TestLoadAssembly", "TestLoadAssembly.dll");
+        static string testtype = "System.Collections.Generic.Dictionary`2[[Program, Foo], [Program, Foo]]";
+        static TypeTestsExtended()
+        {
+            // Move TestLoadAssembly.dll to subfolder TestLoadAssembly
+            try
+            {
+                Directory.CreateDirectory(Path.GetDirectoryName(destTestAssemblyPath));
+                File.Move(sourceTestAssemblyPath, destTestAssemblyPath);
+            }
+            catch (System.Exception) { }
+            finally
+            {
+                File.Delete(sourceTestAssemblyPath);
```

Comment:
Move throws IOException if destination file exists.
So in case of incremental build the source file will exist and we will fail the loadfrom of the file as the source location is part of the TPA list